### ISSUE TITLE
Implement long polling transport

### ISSUE THREAD

**Issue by moozzyk** [SignalR/SignalR-Client-Cpp#62]

**Comment by nicolasr75** [comment]
I have successfully used SignalR-Cpp to connect some C++ applications to an ASP.NET web application with IIS 8. It works really great! It would be even better if I could support some existing installations with IIS 7.5 which does not have web sockets. Are there any plans to support the long polling transport in the near future?

**Comment by moozzyk** [comment]
@nicolasr75 we don't have any specific timeline at the moment.

**Comment by nicolasr75** [comment]
OK. Thanks anyway, I think you did a great job with this project!

**Comment by matt14m** [comment]
Any update on long polling? We've got web sockets doing great things but need to support older OS's / servers.

**Comment by Sonic78** [comment]
Any update on long polling? We know of many scenarios where firewalls will not allow an upgrade of the http connection to long polling so we need the possibility to use a fallback else SignalR is no alternative for us?

**Comment by ghost** [comment]
@moozzyk Do you happen to have a rough estimate of how much effort it would be to support Long Polling? We'd urgently need a native SignalR client with fallback support, and we'd think about implementing it ourselves.

**Comment by moozzyk** [comment]
My rough estimate would be ~2 weeks but I think there might be a few tricky spots.

**Comment by matelich** [comment]
I see some work (https://github.com/aspnet/SignalR-Client-Cpp/commit/4eabfb4429d3a33a8f13063e6683eba54bd1c64a) has been done since your last comment here. I assume that was just a small bit of work against the 2 week estimate?

**Comment by moozzyk** [comment]
Yeah. I started looking at this some time ago but have been busy with other things.

**Comment by a-ilyin** [comment]
Hello, everyone.
I suppose all the code moved to https://github.com/aspnet/SignalR-Client-Cpp and no progress will be done here. 
Am I right?
Any chance long polling transport to be implemented?

**Comment by BrennanConroy** [comment]
This Repo is for **ASP.NET SignalR** and will likely have very little code changes in the future.

Work on the new client for **ASP.NET Core SignalR** is located at the [repo](https://github.com/aspnet/SignalR-Client-Cpp) you linked.
